api/build.gradle.kts

```
@@ -13,6 +13,7 @@ plugins {
 // papermc repo + disableAutoTargetJvm needed for mockbukkit
 repositories {
     maven("https://repo.papermc.io/repository/maven-public/")
+    maven("https://jitpack.io") // for MockBukkit 1.21 (until it's added to Paper's Maven repo)
```

if its only for mockbukkit, limit its content then:
```suggestion
    maven("https://jitpack.io/") {
        content {
            includeGroup("com.github.MockBukkit")
        }
    }
```